lib/rspec/support/recursive_const_methods.rb

```diff
@@ -5,52 +5,27 @@ module Support
     module RecursiveConstMethods
       # We only want to consider constants that are defined directly on a
       # particular module, and not include top-level/inherited constants.
-      # Unfortunately, the constant API changed between 1.8 and 1.9, so
-      # we need to conditionally define methods to ignore the top-level/inherited
-      # constants.
       #
-      # Given:
```

Comment:
I think it reads better if this goes back.

Reply:
Agree, it is non-trivial, and additional verbosity won't hurt.